e2etest/runner.go

```diff
@@ -107,6 +107,9 @@ func (t *TestRunner) SetAllFlags(p params, o Operation) {
 	set("is-object-dir", p.isObjectDir, false)
 	set("debug-skip-files", strings.Join(p.debugSkipFiles, ";"), "")
 	set("check-md5", p.checkMd5.String(), "FailIfDifferent")
+	if p.trailingDot != nil {
```

Comment:
This pattern only really makes sense if the zero-value is not the default; does p.trailingDot need to be a pointer for some reason?

Reply:
This is because I need a way to test that if nothing is passed, the trailing dot files still work.

Reply:
Right; but that's the default behavior. The set() function doesn't apply the parameter if it's default, iirc.

Reply:
changed this to use set's behavior in the correct way